projects/hashflow/index.js

```diff
@@ -18,11 +21,20 @@ function chainTvl(chain) {
   return async (timestamp, ethBlock, chainBlocks) => {
     const balances = {};
     const block = await getBlock(timestamp, chain, chainBlocks);
-    const transformAddress = id=>`${chain}:${id}`;
+    const transformAddress = await getChainTransform(chain);
+    const chainId = chainIds[chain]
+    const chainFile = `${__dirname}/${chainId}.json`
+    const url = `${http_api_url}?networkId=${chainId}&lp=${null_addr}`;
+    let pools_response
 
-    const url = `${http_api_url}?networkId=${chainIds[chain]}`;
-    const pools_response = await retry(async () => await axios.get(url));
-    const pools = pools_response.data.pools.map(pool => 
+    try {
+      pools_response = (await retry(async () => await axios.get(url))).data
```

Comment:
wont this result in inconsistent data?

Reply:
you mean, it would lead to spikes? other options is, we dont call the api at all and manually update our list based on teams update on pool info

Reply:
yeah, if api fails inconsistently every hour we might see a different number, so there will be spikes on chart

Reply:
you are right, hence I removed the api call. We have a fixed pool list now, but we can update it when the team requests?